awx/main/migrations/0188_inbound_hop_nodes.py

```diff
@@ -0,0 +1,106 @@
+# Generated by Django 4.2.6 on 2023-12-14 19:14
```

Comment:
This migration has no data migration.  From what I see currently, we need:

- [ ] create a canonical `ReceptorAddress` when the listener is turned on, with copies of the port, address, and `peers_from_control_plane` (as an aside, does `ReceptorAddress` actually need this field?)
- [ ] existing `InstanceLink` entries need to migrate their `target` data